### ISSUE TITLE
vkcube:  /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol:

### ISSUE THREAD

**Issue by perfect7gentleman** [KhronosGroup/Vulkan-Tools#508]
```
 ~ $ LD_DEBUG=libs vkcube
      6541:     find library=libwayland-client.so.0 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libwayland-client.so.0
      6541:
      6541:     find library=libm.so.6 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/lib64/libm.so.6
      6541:
      6541:     find library=libvulkan.so.1 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libvulkan.so.1
      6541:
      6541:     find library=libgomp.so.1 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/libgomp.so.1
      6541:
      6541:     find library=libc.so.6 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/lib64/libc.so.6
      6541:
      6541:     find library=libffi.so.7 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libffi.so.7
      6541:
      6541:     find library=libpthread.so.0 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/lib64/libpthread.so.0
      6541:
      6541:     find library=libdl.so.2 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/lib64/libdl.so.2
      6541:
      6541:
      6541:     calling init: /lib64/libpthread.so.0
      6541:
      6541:
      6541:     calling init: /lib64/ld-linux-x86-64.so.2
      6541:
      6541:
      6541:     calling init: /lib64/libc.so.6
      6541:
      6541:
      6541:     calling init: /lib64/libdl.so.2
      6541:
      6541:
      6541:     calling init: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/libgomp.so.1
      6541:
      6541:
      6541:     calling init: /usr/lib64/libffi.so.7
      6541:
      6541:
      6541:     calling init: /lib64/libm.so.6
      6541:
      6541:
      6541:     calling init: /usr/lib64/libvulkan.so.1
      6541:
      6541:
      6541:     calling init: /usr/lib64/libwayland-client.so.0
      6541:
      6541:
      6541:     initialize program: vkcube
      6541:
      6541:
      6541:     transferring control: vkcube
      6541:
      6541:     find library=libdrm.so.2 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libdrm.so.2
      6541:
      6541:     find library=libxcb-dri3.so.0 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libxcb-dri3.so.0
      6541:
      6541:     find library=libz.so.1 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/lib64/libz.so.1
      6541:
      6541:     find library=libexpat.so.1 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libexpat.so.1
      6541:
      6541:     find library=libzstd.so.1 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libzstd.so.1
      6541:
      6541:     find library=libxcb.so.1 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libxcb.so.1
      6541:
      6541:     find library=libX11-xcb.so.1 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libX11-xcb.so.1
      6541:
      6541:     find library=libxcb-present.so.0 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libxcb-present.so.0
      6541:
      6541:     find library=libxcb-sync.so.1 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libxcb-sync.so.1
      6541:
      6541:     find library=libxcb-randr.so.0 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libxcb-randr.so.0
      6541:
      6541:     find library=libxshmfence.so.1 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libxshmfence.so.1
      6541:
      6541:     find library=libstdc++.so.6 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/libstdc++.so.6
      6541:
      6541:     find library=libgcc_s.so.1 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/libgcc_s.so.1
      6541:
      6541:     find library=libXau.so.6 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libXau.so.6
      6541:
      6541:     find library=libXdmcp.so.6 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libXdmcp.so.6
      6541:
      6541:     find library=libbsd.so.0 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libbsd.so.0
      6541:
      6541:     find library=libmd.so.0 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/usr/lib64/libmd.so.0
      6541:
      6541:
      6541:     calling init: /usr/lib64/libmd.so.0
      6541:
      6541:
      6541:     calling init: /usr/lib64/libbsd.so.0
      6541:
      6541:
      6541:     calling init: /usr/lib64/libXdmcp.so.6
      6541:
      6541:
      6541:     calling init: /usr/lib64/libXau.so.6
      6541:
      6541:
      6541:     calling init: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/libgcc_s.so.1
      6541:
      6541:
      6541:     calling init: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/libstdc++.so.6
      6541:
      6541:
      6541:     calling init: /usr/lib64/libxshmfence.so.1
      6541:
      6541:
      6541:     calling init: /usr/lib64/libxcb.so.1
      6541:
      6541:
      6541:     calling init: /usr/lib64/libxcb-randr.so.0
      6541:
      6541:
      6541:     calling init: /usr/lib64/libxcb-sync.so.1
      6541:
      6541:
      6541:     calling init: /usr/lib64/libxcb-present.so.0
      6541:
      6541:
      6541:     calling init: /usr/lib64/libX11-xcb.so.1
      6541:
      6541:
      6541:     calling init: /usr/lib64/libzstd.so.1
      6541:
      6541:
      6541:     calling init: /usr/lib64/libexpat.so.1
      6541:
      6541:
      6541:     calling init: /lib64/libz.so.1
      6541:
      6541:
      6541:     calling init: /usr/lib64/libxcb-dri3.so.0
      6541:
      6541:
      6541:     calling init: /usr/lib64/libdrm.so.2
      6541:
      6541:
      6541:     calling init: /usr/lib64/libvulkan_intel.so
      6541:
MESA-INTEL: warning: Haswell Vulkan support is incomplete
      6541:     find library=libnss_files.so.2 [0]; searching
      6541:      search cache=/etc/ld.so.cache
      6541:       trying file=/lib64/libnss_files.so.2
      6541:
      6541:
      6541:     calling init: /lib64/libnss_files.so.2
      6541:
      6541:     /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol: _nss_files_getcanonname_r (fatal)
      6541:     /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol: _nss_files_gethostbyaddr2_r (fatal)
      6541:     /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol: _nss_files_getpublickey (fatal)
      6541:     /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol: _nss_files_getsecretkey (fatal)
      6541:     /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol: _nss_files_netname2user (fatal)
Selected GPU 0: Intel(R) HD Graphics 4600 (HSW GT2), type: 1
Segmentation fault
```

**Comment by perfect7gentleman** [comment]
>Do other Vulkan programs (vulkaninfo?) run without issues?

Vulkaninfo runs fine. But other programs (mpv, for example) using vulkan fail with the same error.
>Has vkcube ever run successfully on this system?
 
Yes
>Have you recently updated your Intel Vulkan driver?

Yes, I use mesa from git.

**Comment by perfect7gentleman** [comment]
The same error on AMD KAVERI
```
 ~ $ LD_DEBUG=libs vkcube
      9996:     find library=libwayland-client.so.0 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libwayland-client.so.0
      9996:
      9996:     find library=libm.so.6 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/lib64/libm.so.6
      9996:
      9996:     find library=libvulkan.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libvulkan.so.1
      9996:
      9996:     find library=libgomp.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/libgomp.so.1
      9996:
      9996:     find library=libpthread.so.0 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/lib64/libpthread.so.0
      9996:
      9996:     find library=libc.so.6 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/lib64/libc.so.6
      9996:
      9996:     find library=libffi.so.7 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libffi.so.7
      9996:
      9996:     find library=libdl.so.2 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/lib64/libdl.so.2
      9996:
      9996:
      9996:     calling init: /lib64/libpthread.so.0
      9996:
      9996:
      9996:     calling init: /lib64/ld-linux-x86-64.so.2
      9996:
      9996:
      9996:     calling init: /lib64/libc.so.6
      9996:
      9996:
      9996:     calling init: /lib64/libdl.so.2
      9996:
      9996:
      9996:     calling init: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/libgomp.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libffi.so.7
      9996:
      9996:
      9996:     calling init: /lib64/libm.so.6
      9996:
      9996:
      9996:     calling init: /usr/lib64/libvulkan.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libwayland-client.so.0
      9996:
      9996:
      9996:     initialize program: vkcube
      9996:
      9996:
      9996:     transferring control: vkcube
      9996:
      9996:     find library=libLLVM-12libcxx.so [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib/llvm/12/lib64/libLLVM-12libcxx.so
      9996:
      9996:     find library=libdrm_amdgpu.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libdrm_amdgpu.so.1
      9996:
      9996:     find library=libelf.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libelf.so.1
      9996:
      9996:     find library=libxcb-dri3.so.0 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libxcb-dri3.so.0
      9996:
      9996:     find library=libz.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/lib64/libz.so.1
      9996:
      9996:     find library=libexpat.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libexpat.so.1
      9996:
      9996:     find library=libzstd.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libzstd.so.1
      9996:
      9996:     find library=libxcb.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libxcb.so.1
      9996:
      9996:     find library=libX11-xcb.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libX11-xcb.so.1
      9996:
      9996:     find library=libxcb-present.so.0 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libxcb-present.so.0
      9996:
      9996:     find library=libxcb-sync.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libxcb-sync.so.1
      9996:
      9996:     find library=libxcb-randr.so.0 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libxcb-randr.so.0
      9996:
      9996:     find library=libxshmfence.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libxshmfence.so.1
      9996:
      9996:     find library=libdrm.so.2 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libdrm.so.2
      9996:
      9996:     find library=libc++.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libc++.so.1
      9996:
      9996:     find library=libc++abi.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libc++abi.so.1
      9996:
      9996:     find library=libunwind.so.1 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libunwind.so.1
      9996:
      9996:     find library=libtinfo.so.6 [0]; searching
      9996:      search path=/usr/lib/llvm/12/lib64/../lib64/glibc-hwcaps/x86-64-v2:/usr/lib/llvm/12/lib64/../lib64/tls/x86_64/x86_64:/usr/lib/llvm/12/lib64/../lib64/tls/x86_64:/usr/lib/llvm/12/lib64/../lib64/tls/x86_64:/usr/lib/llvm/12/lib64/../lib64/tls:/usr/lib/llvm/12/lib64/../lib64/x86_64/x86_64:/usr/lib/llvm/12/lib64/../lib64/x86_64:/usr/lib/llvm/12/lib64/../lib64/x86_64:/usr/lib/llvm/12/lib64/../lib64               (RUNPATH from file /usr/lib/llvm/12/lib64/libLLVM-12libcxx.so)
      9996:       trying file=/usr/lib/llvm/12/lib64/../lib64/glibc-hwcaps/x86-64-v2/libtinfo.so.6
      9996:       trying file=/usr/lib/llvm/12/lib64/../lib64/tls/x86_64/x86_64/libtinfo.so.6
      9996:       trying file=/usr/lib/llvm/12/lib64/../lib64/tls/x86_64/libtinfo.so.6
      9996:       trying file=/usr/lib/llvm/12/lib64/../lib64/tls/x86_64/libtinfo.so.6
      9996:       trying file=/usr/lib/llvm/12/lib64/../lib64/tls/libtinfo.so.6
      9996:       trying file=/usr/lib/llvm/12/lib64/../lib64/x86_64/x86_64/libtinfo.so.6
      9996:       trying file=/usr/lib/llvm/12/lib64/../lib64/x86_64/libtinfo.so.6
      9996:       trying file=/usr/lib/llvm/12/lib64/../lib64/x86_64/libtinfo.so.6
      9996:       trying file=/usr/lib/llvm/12/lib64/../lib64/libtinfo.so.6
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/lib64/libtinfo.so.6
      9996:
      9996:     find library=libXau.so.6 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libXau.so.6
      9996:
      9996:     find library=libXdmcp.so.6 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libXdmcp.so.6
      9996:
      9996:     find library=libbsd.so.0 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libbsd.so.0
      9996:
      9996:     find library=libmd.so.0 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/usr/lib64/libmd.so.0
      9996:
      9996:
      9996:     calling init: /usr/lib64/libmd.so.0
      9996:
      9996:
      9996:     calling init: /usr/lib64/libbsd.so.0
      9996:
      9996:
      9996:     calling init: /usr/lib64/libXdmcp.so.6
      9996:
      9996:
      9996:     calling init: /usr/lib64/libXau.so.6
      9996:
      9996:
      9996:     calling init: /lib64/libtinfo.so.6
      9996:
      9996:
      9996:     calling init: /usr/lib64/libunwind.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libc++abi.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libc++.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libdrm.so.2
      9996:
      9996:
      9996:     calling init: /usr/lib64/libxshmfence.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libxcb.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libxcb-randr.so.0
      9996:
      9996:
      9996:     calling init: /usr/lib64/libxcb-sync.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libxcb-present.so.0
      9996:
      9996:
      9996:     calling init: /usr/lib64/libX11-xcb.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libzstd.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libexpat.so.1
      9996:
      9996:
      9996:     calling init: /lib64/libz.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libxcb-dri3.so.0
      9996:
      9996:
      9996:     calling init: /usr/lib64/libelf.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib64/libdrm_amdgpu.so.1
      9996:
      9996:
      9996:     calling init: /usr/lib/llvm/12/lib64/libLLVM-12libcxx.so
      9996:
      9996:
      9996:     calling init: /usr/lib64/libvulkan_radeon.so
      9996:
      9996:     find library=libnss_files.so.2 [0]; searching
      9996:      search cache=/etc/ld.so.cache
      9996:       trying file=/lib64/libnss_files.so.2
      9996:
      9996:
      9996:     calling init: /lib64/libnss_files.so.2
      9996:
      9996:     /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol: _nss_files_getcanonname_r (fatal)
      9996:     /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol: _nss_files_gethostbyaddr2_r (fatal)
      9996:     /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol: _nss_files_getpublickey (fatal)
      9996:     /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol: _nss_files_getsecretkey (fatal)
      9996:     /lib64/libnss_files.so.2: error: symbol lookup error: undefined symbol: _nss_files_netname2user (fatal)
WARNING: radv is not a conformant vulkan implementation, testing use only.
Selected GPU 0: AMD RADV KAVERI (ACO), type: 1
Segmentation fault
```

**Comment by Tony-LunarG** [comment]
Ok, so it's probably not the driver (even though RADV and Intel are both Mesa drivers).  It looks to me like libnss_files.so is part of glibc - have you installed a newer version of glibc recently?  If so, is it possible to go back to your previous version?

**Comment by perfect7gentleman** [comment]
Glibc-2.33 was installed in February. Only GCC-11.1 was installed at the end of April.

**Comment by MarkY-LunarG** [comment]
A few things:

1) Libnss_files is part of ssh.  Try installing that.

2) According to the web, it seems like Haswell 4600 doesn't support Vulkan. https://www.intel.com/content/www/us/en/support/articles/000005524/graphics.html, look under "Intel Core Processors".

3) For the Kavari, some people have luck with the AMDVLK driver.

Maybe one of those things will help.

**Comment by MarkY-LunarG** [comment]
Also, according to [this post](https://github.com/oracle/graal/issues/3215) (grain of salt required) glib-c 2.33 has bad symbol info and you should use `LD_DEBUG=unused`.  Not sure where to apply that, but you can try reading the link.

**Comment by Tony-LunarG** [comment]
Also, could we get a stack trace for the segfault you are seeing?

**Comment by perfect7gentleman** [comment]
@MarkY-LunarG 
1. They are part of Glibc
 ``` ~ $ equery b /lib64/libnss_files.so.2 
 * Searching for /lib64/libnss_files.so.2 ... 
sys-libs/glibc-2.33 (/lib64/libnss_files-2.33.so)
sys-libs/glibc-2.33 (/lib64/libnss_files.so.2 -> libnss_files-2.33.so)```
2. Haswell supports Vulkan, but not completely.

I tried LD_DEBUG=unused, it didn't work in that case.

@Tony-LunarG , you mean strace?

**Comment by perfect7gentleman** [comment]
LD_DEBUG=all vkcube - https://gist.github.com/perfect7gentleman/5edf712109bc20b9cc3beed28f3a5db9

strace vkcube - https://gist.github.com/perfect7gentleman/81b8bf032d679667d9e0671e655bc6be

**Comment by MarkY-LunarG** [comment]
In the strace, it's still trying to use the Haswell which isn't supposed to have support.  Just curious, what if you turn on `VK_LOADER_DEBUG=all` as well (warning it will spew too)?  I'm curious if it's failing a dynamic symbol lookup somewhere, or maybe it's still accessing your Haswell thinking it can do Vulkan even when you're trying to use your discrete card.

**Comment by perfect7gentleman** [comment]
dmesg showed new error
```
[Fri May 21 09:42:38 2021] vkcube[7770]: segfault at 0 ip 00007f05547b0e78 sp 00007ffe50a205f0 error 4 cpu 2 in libwayland-client.so.0.3.0[7f05547ae000+6000]
[Fri May 21 09:42:38 2021] Code: 9c 24 20 01 00 00 c5 f8 29 a4 24 30 01 00 00 c5 f8 29 ac 24 40 01 00 00 c5 f8 29 b4 24 50 01 00 00 c5 f8 29 bc 24 60 01 00 00 <49> 8b 13 48 8d 84 24 90 01 00 00 4c 8d 6c 24 20 48 89 e1 48 89 44
[Fri May 21 09:43:23 2021] vkcube[9960]: segfault at 0 ip 00007fa340c06e78 sp 00007ffe82be5dd0 error 4 cpu 0 in libwayland-client.so.0.3.0[7fa340c04000+6000]
[Fri May 21 09:43:23 2021] Code: 9c 24 20 01 00 00 c5 f8 29 a4 24 30 01 00 00 c5 f8 29 ac 24 40 01 00 00 c5 f8 29 b4 24 50 01 00 00 c5 f8 29 bc 24 60 01 00 00 <49> 8b 13 48 8d 84 24 90 01 00 00 4c 8d 6c 24 20 48 89 e1 48 89 44
[Fri May 21 09:44:08 2021] vkcube[11773]: segfault at 0 ip 00007f935f8ab31b sp 00007fff1c9ccf60 error 4 cpu 3 in libwayland-client.so.0.3.0[7f935f8aa000+6000]
[Fri May 21 09:44:08 2021] Code: 9c 24 20 01 00 00 c5 f8 29 a4 24 30 01 00 00 c5 f8 29 ac 24 40 01 00 00 c5 f8 29 b4 24 50 01 00 00 c5 f8 29 bc 24 60 01 00 00 <48> 8b 55 00 4c 8d 74 24 20 48 8d 84 24 a0 01 00 00 c7 44 24 08 18
[Fri May 21 09:44:39 2021] vkcube[13786]: segfault at 0 ip 00007fe227899e88 sp 00007ffecb799a90 error 4 cpu 0 in libwayland-client.so.0.3.0[7fe227897000+6000]
[Fri May 21 09:44:39 2021] Code: 9c 24 20 01 00 00 c5 f8 29 a4 24 30 01 00 00 c5 f8 29 ac 24 40 01 00 00 c5 f8 29 b4 24 50 01 00 00 c5 f8 29 bc 24 60 01 00 00 <49> 8b 13 48 8d 84 24 90 01 00 00 4c 8d 6c 24 20 48 89 e1 48 89 44
```

**Comment by perfect7gentleman** [comment]
Fixed in 1.2.181.

**Comment by Tony-LunarG** [comment]
The only thing I can see that might have fixed this is #527 Thanks for the report and the followup
